File: AD-HOC/1542/main.js
function totalPages(q, d, p) {
    return parseInt((q * d * p) / (p - q));
}

function formattedPrint(pages) {
    console.log(`${pages} ${pages > 1 ? 'paginas' : 'pagina'}`);
}

let lines = require('fs').readFileSync('/dev/stdin', 'utf-8').split('\n');
let quantities = lines.shift().split(' ').map(Number);
while (quantities.length > 1) {
    let pages = totalPages(...quantities);
    formattedPrint(pages);
    quantities = lines.shift().split(' ').map(Number);
}

File: AD-HOC/1542/main.c
#include <stdio.h>

int totalPages(int q, int d, int p) {
    return (q * d * p) / (p - q);
}

void formattedPrint(int pages) {
    printf("%d %s\n", pages, pages > 1 ? "paginas" : "pagina");
}

int main() {
    int q, d, p;
    while (scanf("%d", &q) && q > 0) {
        scanf("%d %d", &d, &p);
        int pages = totalPages(q, d, p);
        formattedPrint(pages);
    }
    return 0;
}
